src/validators/function.rs

```diff
@@ -303,3 +297,23 @@ pub fn convert_err<'a>(py: Python<'a>, err: PyErr, input: &'a impl Input<'a>) ->
         ValError::InternalErr(err)
     }
 }
+
+#[pyclass(module = "pydantic_core._pydantic_core")]
+pub struct ValidationInfo {
+    #[pyo3(get)]
+    data: Option<Py<PyDict>>,
+    #[pyo3(get)]
+    config: PyObject,
+    #[pyo3(get)]
+    context: Option<PyObject>,
+}
+
+impl ValidationInfo {
+    fn new(extra: &Extra, config: &PyObject, py: Python) -> Self {
```

Comment:
by convention `py` is generally the first parameter when used.

I have a fix for this ready to fix, but I think you haven't allowed allowed maintainers to push to your PR.

Reply:
once this is done, this is ready to merge.

Reply:
ignore me, this is already merged - still too early for me 🤦.

I'll push to tweak to main.

Reply:
This was already merged. Is this a PyO3 convention or our convention? As far as I can tell in PyO3 it's used as both the first and last.

Reply:
I thought pyo3, maybe I'm wrong.

Definitely our convention.